### PR TITLE
Simplify example schema plugins

### PR DESCRIPTION
**Pull request by devknoll**
Same as https://github.com/facebook/relay-starter-kit/pull/2

### CODE REVIEW

File: examples/relay-treasurehunt/build/babelGraphQLPlugin.js [hunk 1]
```diff
@@ -1,9 +1,4 @@
-var fs = require('fs');
-var getBabelRelayPlugin = require('babel-relay-plugin');
-var path = require('path');
+var getBabelGraphQLPlugin = require('babel-relay-plugin');
```
Comment (voideanvalue) [inline examples/relay-treasurehunt/build/babelGraphQLPlugin.js]: Do we want to use `BabelRelayPlugin` and `BabelGraphQLPlugin` interchangeably? cc @steveluscher @yungsters

Comment (devknoll) [inline examples/relay-treasurehunt/build/babelGraphQLPlugin.js]: Good point. I'll change it to getBabelRelayPlugin since that's what it's actually called.

Comment (devknoll) [inline examples/relay-treasurehunt/build/babelGraphQLPlugin.js]: Though, that doesn't help the files being named `babelGraphQLPlugin.js`...

Comment (voideanvalue) [inline examples/relay-treasurehunt/build/babelGraphQLPlugin.js]: Also, the file is named `babelGraphQLPlugin.js` :stuck_out_tongue_closed_eyes:

Comment (devknoll) [inline examples/relay-treasurehunt/build/babelGraphQLPlugin.js]: Ok, everything is `babelRelayPlugin` now ;-)

**Comment by voideanvalue** [comment]
Awesome. Thank you!

**Pull request merged**